azext_edge/tests/edge/init/int/test_init_int.py

```diff
@@ -0,0 +1,117 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+import pytest
+from os import mkdir
+
+from ....helpers import run
+from .dataprocessor_helper import assert_dataprocessor_args
+from .mq_helper import assert_mq_args
+from .opcua_helper import assert_simulate_plc_args
+from .orchestrator_helper import assert_orchestrator_args
+from .helper import assert_init_result
+
+
+@pytest.fixture(scope="function")
+def init_test_setup(cluster_connection, settings):
+    from ....settings import EnvironmentVariables
+    settings.add_to_config(EnvironmentVariables.rg.value)
+    settings.add_to_config(EnvironmentVariables.kv.value)
+    settings.add_to_config(EnvironmentVariables.cluster.value)
+    settings.add_to_config(EnvironmentVariables.sp_app_id.value)
+    settings.add_to_config(EnvironmentVariables.sp_object_id.value)
+    settings.add_to_config(EnvironmentVariables.sp_secret.value)
+    settings.add_to_config(EnvironmentVariables.init_args.value)
+    settings.add_to_config(EnvironmentVariables.aio_cleanup.value)
+
+    if not all([settings.env.azext_edge_cluster, settings.env.azext_edge_rg, settings.env.azext_edge_kv]):
+        raise AssertionError(
+            "Cannot run init tests without a connected cluster, resource group, and precreated keyvault. "
+            f"Current settings: {settings}"
+        )
+    yield {
+        "clusterName": settings.env.azext_edge_cluster,
+        "resourceGroup": settings.env.azext_edge_rg,
+        "keyVault": settings.env.azext_edge_kv,
+        "servicePrincipalAppId": settings.env.azext_edge_sp_app_id,
+        "servicePrincipalObjectId": settings.env.azext_edge_sp_object_id,
+        "servicePrincipalSecret": settings.env.azext_edge_sp_secret,
+        "additionalArgs": settings.env.azext_edge_init_args.strip('"')
+    }
+    if settings.env.azext_edge_aio_cleanup:
+        run(
+            f"az iot ops delete --cluster {settings.env.azext_edge_cluster} -g {settings.env.azext_edge_rg} "
+            "-y --no-progress --force"
+        )
+
+
+@pytest.mark.init_scenario_test
```

Comment:
This warning can be eliminated.

![image](https://github.com/Azure/azure-iot-ops-cli-extension/assets/28658112/970e0909-dfff-411d-ba02-b2c2cf25221f)